Kruskals/kruskals.py

```diff
@@ -65,15 +66,18 @@ def driver_score(self, directional=False):
             fact = factorial(ind_c - 1) / (2 * factorial(ind_c - 3))
             pijm_row_mean = np.nanmean(pijm, axis=(0, 2)) * fact
             self._driver_score = (pij_row_mean + pijm_row_mean) / ((ind_c - 1) + fact)
-            if directional:
-                self._driver_score = self._driver_score * np.apply_along_axis(self.correlation_coef, 0, self._ndarr, self._arr)
-        return np.nan_to_num(self._driver_score)
+            self._driver_score = np.nan_to_num(self._driver_score)
+        if directional:
+            self._driver_score = self._driver_score * np.apply_along_axis(self.correlation_coef, 0, self._ndarr, self._arr)
+        if percentage:
+            return self._driver_score / np.fabs(self._driver_score).sum() * 100
+        else:
+            return self._driver_score
 
-    def percentage(self):
-        """
-        Distance as a relative percentage
-        """
-        return self.distance() / self.distance().sum() * 100
+    def percentage(self, directional=False):
+        """ Distance as a relative percentage """
```

Comment:
Because:

``` python
k = Kruskals(ndarr, arr)
k.percentage(True) == k.percentage(False) # because caching
```
I could make it so it checks the whether the driver score was cached for which direction, or move the directional stuff out. One makes it more complicated initial if block that was gross and the other meant that it would be doing calculations every time you ran percentage. Hence moved it inside. Let me know your thoughts @xulaus